### DISCUSSION TITLE
Suggestion: If parentheses are transformed in pairs, move the input position inside the parentheses.

### DISCUSSION THREAD

**Discussion by mikunimaru** [google/mozc#887] [Ideas]
When outputting parentheses in pairs, most users type characters inside the parentheses.
I think it would be convenient if mozc moved the input position to the inside of the parentheses when the parentheses were output in pairs, like the auto-completion of half-width parentheses in VSCode.
Useful if the user wants to pre-output just the parenthesis pair to avoid forgetting to close it.

There are two ideas that can be combined.
Idea 1. The input position moves to the middle of the parentheses when the user converts the parentheses in pairs alone. "かっこ"→"「|」","（）"→"【|】"
Idea 2. If the user converts with "かっこあいう", it will be converted to be sandwiched like "「あいう」".

(In the case of Idea 2, the user can convert "（）あいう" to "「あいう」","（あいう）","【あいう】",etc.)

**Comment by hiroyuki-komatsu** [comment]
Hi mikunimaru,

Indeed, https://github.com/google/mozc/commit/e8cf4194d8b8c78d10ff61941d881c54a85da7af is the commit for that functionality. We need to update the client side too.

Thanks,
